src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -178,6 +179,17 @@ public boolean hasBrewingRecipes() {
 		return false;
 	}
 
+	public boolean hasGameRulesOfType(String type) {
+		for (ModElement element : workspace.getModElements())
+			if (element.getType() == ModElementType.GAMERULE) {
+				GeneratableElement ge = element.getGeneratableElement();

```

Comment:
You don't need anymore to declare this variable.

Reply:
:heavy_check_mark: Fixed